### ISSUE TITLE
Batch Normalization

### ISSUE THREAD

**Issue by zhengyang-wang** [zhengyang-wang/Unet_3D#3]
It seems that the BN layer has some problems for 3D cases. It does not work as I expected, which may affect the accuracy. I'll try to fix it.

**Comment by zhengyang-wang** [comment]
During inference, the BN layer should take is_training=false to use moving_mean and moving_variance. Based on current code, I suggest to change utils/ops.py manually by adding the argument 'is_training=false' to every 'tf.contrib.layers.batch_norm' when using '--option=predict'.
